COLLABORATOR_GUIDE.md

```diff
@@ -111,10 +111,10 @@ comment that explains why the PR does not require a CI run.
 
 ### Code Reviews
 
-All pull requests must be reviewed and accepted by a Collaborator with
-sufficient expertise who is able to take full responsibility for the
-change. In the case of pull requests proposed by an existing
-Collaborator, an additional Collaborator is required for sign-off.
+All pull requests must be reviewed and accepted by at least two Collaborators
+with sufficient expertise who are able to take full responsibility for the
+change. Approval must be from Collaborators who are not authors of the proposed
```

Comment:
Perhaps "authors" is a bit vague here? Does that mean the individual who opened the PR, anyone that contributed a commit to the PR, anyone that requested the changes that led to the opening of the PR, or something else (including a combination of the above)?